operator/config/crd/bases/forklift.konveyor.io_forkliftcontrollers.yaml

```
@@ -32,7 +32,356 @@ spec:
+              # Source file is: operator/config/crd/bases/forklift.konveyor.io_forkliftcontrollers.yaml,
+              # No need to edit other files, other files containing the CRD definitions are automatically updated.
+              #
+              # NOTE: This list was auto generated by an AI agent
+
+              # Feature Gates
+              feature_ui_plugin:
+                type: string
+                enum: ["true", "false"]
+                description: "Enable UI plugin (default: true)"
+              feature_validation:
+                type: string  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)